internal/cli/project/create.go

```diff
@@ -0,0 +1,42 @@
+package project
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/bufbuild/connect-go"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/akuity/kargo/internal/cli/option"
+	v1alpha1 "github.com/akuity/kargo/pkg/api/service/v1alpha1"
+	"github.com/akuity/kargo/pkg/api/service/v1alpha1/svcv1alpha1connect"
+)
+
+func newCreateCommand(opt *option.Option) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "create",
+		Args:    cobra.ExactArgs(1),
+		Example: "kargo project create (NAME)",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+
+			name := strings.TrimSpace(args[0])
```

Comment:
Is there some other more canonical way of retrieving positional arguments in cobra?